### DISCUSSION TITLE
Tooltip on disabled segmentedcontroll

### DISCUSSION THREAD

**Discussion by ofirgo1** [mantinedev/mantine#6727] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

6

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

None

### Describe the bug

I want to add tooltip to disabled segmented control and the tooltip didn't appear.
I would like to know if there is a solution for this problem.

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
See example with disabled button and tooltip – https://mantine.dev/core/button/#disabled-button-with-tooltip

**Reply by ofirgo1** [reply]
I don't understand that...
There is no data-disabled prop in the segment control or the segment control data props.
